### ISSUE TITLE
Sectioned. Tabs cannot be changed on tests

### ISSUE THREAD

**Issue by ghost** [HGustavs/LenaSYS#7577]
![image](https://user-images.githubusercontent.com/49199825/58418254-93caea00-8087-11e9-806f-bebd5bd396da.png)

![image](https://user-images.githubusercontent.com/49199825/58418318-c96fd300-8087-11e9-8612-1509d8e8bb06.png)

Trying to change the Tabs for a test. But the test never change

**Comment by a18mahkh** [comment]
If you create a new 'Moment' then you don't need tabs on 'Test' for indentation. I think this moment option is there to separate tests??
![Anteckning 2020-04-16 112901](https://user-images.githubusercontent.com/49142349/79439746-7d968700-7fd5-11ea-98be-ef5ca2dae378.png)

@HGustavs @mbrohede

**Comment by a17towan** [comment]
It is my of my understanding that "tabs" is supposed to indentate the circles (the types) further to the right depending on how high of a number you pick. If this is the case then it would seem that "test", or "duggor" already have indentation so they naturally align themselves slightly to the right when you create them; regardless if they are below a "moment" or not. 

It looks fine to me, so personally I'm not sure if the "tabs" option is needed. But I could be wrong? @HGustavs @a97marbr

**Comment by a97marbr** [comment]
A moment is the equivalent of a "delkurs". For example, in the project course we have `Projekt  10,0 hp`, ` Seminarier och individuell rapport  2,0 hp` and `Hemtentamen 3,0 hp`

The discussion if having a dugga that is not part of a moment is a valid placement has not been decided. I know that @HGustavs and myself has had this discussion some years ago.

However,
What if we want to split an assignment into several steps with individual deadlines? Let's say for example, that a course has two assignments in one moment and each assignment has several tasks with individual deadlines?

* Moment
  * Assignment 1
    * Assignment 1 - Subtask 1
    * Assignment 1 - Subtask 2
    * Assignment 1 - Subtask 3
  * Assignmane 2
    * Assignment 2 - Subtask 1
    * Assignment 2 - Subtask 2
    * Assignment 2 - Subtask 3

This we cannot visible show at the moment, right?

**Comment by a17towan** [comment]
Alright yes, then I understand why tabs is important for tests! Good explanation.

**Comment by a17towan** [comment]
@a97marbr 

Curious, what are "1 tab + end" and "2 tabs + end" supposed to do? What position should the dugga be moved to? 

![image](https://user-images.githubusercontent.com/49142028/80218044-f08ca700-8640-11ea-819c-1321494d1921.png)

**Comment by a17towan** [comment]
This is how I have progressed so far:
![image](https://user-images.githubusercontent.com/49142028/80364739-b280c480-8886-11ea-93c6-96788b15305c.png)
Each dugga in this image has a different tab value to show the differences.

Which leaves me wondering, if I have interpreted this issue correctly, what "1 tab + end" and "2 tabs + end" are supposed to do. @a97marbr 

Is it just to further indentate to the right?

**Comment by a17towan** [comment]
This is the newest change made.
![image](https://user-images.githubusercontent.com/49142028/80576580-bb4bd480-8a05-11ea-92f8-132f45850e58.png)

@a97marbr Is this what you wanted?

**Comment by a97marbr** [comment]
When indenting a new vertical line should be added. Each level of indentation should have a vertical line.

The end options should have a curved line pointing to the right for its indentation level.

**Comment by a17towan** [comment]
So would it be like this? 
![image](https://user-images.githubusercontent.com/49142028/80948208-f24d2c00-8df1-11ea-85d9-d3ad9451296d.png)

Or this?
![image](https://user-images.githubusercontent.com/49142028/80948362-3809f480-8df2-11ea-814c-733a278b1a35.png)

Or this even?
![image](https://user-images.githubusercontent.com/49142028/80948490-815a4400-8df2-11ea-92ad-daa1d89adbf0.png)


@a97marbr

**Comment by a97marbr** [comment]
The name decides number of vertical bars

1-tab : one vertical bar
2-tabs : two vertical bars
3-tabs : three vertical bars
etc.

1-tab-end : one vertical bar followed by curved end
2-tabs-end : two vertical bars followed by curved end
etc.

Perhaps we need more variants?
end : curved end at tab-0
end-end : End tab-0 and tab-1
end-end-end : end tab-0, tab-1 and tab-2
1-tab-end-end
2-tabs-end
...

I would not have more than 3 levels of indent ... I think

**Comment by a17towan** [comment]
I'm trying my best to interpret this, haha. 

So, I've been thinking, and I went back to the real and actual LenaSYS page where I found this example on the course page for "Webbutveckling - Webbplatsdesign":
![image](https://user-images.githubusercontent.com/49142028/81061324-6eb83b80-8ed4-11ea-812f-4d5389f4735e.png)

If I've understood at least one part correctly: 

1. The "**Dugga: First published webpage**" has "tabs 0", which is just default. 
2. The "**Example: Organize your files**" code page has the tabs "end" which can also be called "0 tabs + end", as in, default end. 

My understanding is: If a dugga that has a default indentation (tabs 0) does not have an 'end' item somewhere beneath it, **there should be no vertical line**. If we add the tabs "end" to one of the items below the highest dugga in the list, there will be a vertical line with a curved end towards the item that has the indentation "end".

So yes, we have:
[0 tabs] = default indentation.
[end] = curved line for [tabs 0].

However, I've noticed something on the **actual** LenaSYS website that differs from the one we are working on in this course, and that is that the "code" items aren't indented.

This is what it looks like on the LenaSYS  that we're working on in the course:
![image](https://user-images.githubusercontent.com/49142028/81063174-a4aaef00-8ed7-11ea-9c47-3046ff3049b9.png)

The "New code" seems to break the intial vertical line that stems from the moment itself. Is this perhaps a new issue? The newly added code should probably end up beneath the "parent" dugga and not break the "moment" line? 

And this brings me to a new question, how should the curved line work with duggas? Because duggas, "tests", have that little circle icon next to them. This would make it slightly awkward for a curved line to take place. This is what I mean:

![image](https://user-images.githubusercontent.com/49142028/81063969-16376d00-8ed9-11ea-9722-313a39b11625.png)

The "end" dugga can't be on the same indentation level as the "parent" dugga because then there can be no curved line. This would mean that we have to indent the "0 tabs end" dugga to the right, but that would place it on the same tabs level as the normal "1 tabs" duggas. This would also mean that we have to indent ALL duggas that are supposed to have an "end" indentation, which would leave us with way more than just 3 indentation levels, like you wanted.

So my question is, should we be able to place "end" tabbings on "duggas", or just on "codes"? Then we first have to fix "codes" so that they don't break the "moments" vertical line. 

Or we can do something like this:

![image](https://user-images.githubusercontent.com/49142028/81065309-82b36b80-8edb-11ea-9adc-48210aeeb06a.png)

What are your opinions on this? @a97marbr
